app/src/main/java/com/wpi/audiojournal/MainActivity.kt

```diff
@@ -9,11 +9,24 @@ import androidx.compose.material.Surface
 import androidx.compose.ui.Modifier
 import androidx.navigation.compose.rememberNavController
 import com.wpi.audiojournal.navigation.SetupNavGraph
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.wpi.audiojournal.models.CategoriesDTO
+import com.wpi.audiojournal.models.MenuItem
+import com.wpi.audiojournal.repositories.AudioJournalService
 import com.wpi.audiojournal.ui.theme.AudioJournalTheme
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
 data class MenuItem(val title: String)
```

Comment:
is this redudant now since there's a MenuItem in the models folder?

Reply:
Yeah I thought I removed this